src/redux/selectors/index.js

```diff
@@ -251,27 +251,35 @@ export const selectedRecordsGroupedByTypeSelector = createSelector(
       return [];
     }
 
-    return sortedGroupedRecordsByType(items)
+    return sortedGroupedRecordsByType({ records: items, descSortOnly: true })
       .filter((group) => group.type === selectedResourceType);
   },
 );
 
 const savedItemsSelector = createSelector(
-  [resourcesSelector, activeCollectionResourceIdsSelector],
-  (resources, saved) => values(resources)
-    .filter(({ id }) => saved[id])
-    .map(pickTimelineFields)
-    .sort(sortByDate),
+  [resourcesSelector, activeCollectionResourceIdsSelector, activeCollectionSelector],
+  (resources, saved, collection) => {
+    const { records } = collection;
+    return values(resources)
+      .filter(({ id }) => saved[id])
+      .map(pickTimelineFields)
+      .map((resource) => {
+        const updatedResource = { ...resource };
```

Comment:
* since `.map(pickTimelineFields)`, creating a new `updatedResource` is unecessary.